Resources/public/js/views/services/plugins/ez-dashboardblockallcontentserviceplugin.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblockallcontentserviceplugin', function (Y) {
+    'use strict';
+
+    /**
+     * Provides the all content dashboard block service plugin class
+     *
+     * @module ez-dashboardblockallcontentserviceplugin
+     */
+    Y.namespace('eZ.Plugin');
+
+    /**
+     * Provides the all content dashboard block service plugin
+     *
+     * @namespace eZ.Plugin
+     * @class DashboardBlockAllContentService
+     * @constructor
+     * @extends eZ.Plugin.ViewServiceBase
+     */
+    Y.eZ.Plugin.DashboardBlockAllContentService = Y.Base.create('dashboardBlockAllContentServicePlugin', Y.eZ.Plugin.ViewServiceBase, [], {
+        initializer: function () {
+            this.onHostEvent('*:getAllContent', this._getAllContent, this);
+        },
+
+        /**
+         * Gets a list of all content
+         *
+         * @method _getAllContent
+         * @protected
+         * @param event {Object} event facade
+         */
+        _getAllContent: function (event) {
+            var host = this.get('host');
+
+            host.fire('contentSearch', {
+                viewName: 'all-content',
+                loadContentType: true,
+                search: {
+                    criteria: {SubtreeCriterion: host.get('rootLocation').get('pathString')},
+                    sortClauses: {DateModifiedClause: 'DESC'},
```

Comment:
for please leave that commented out and add the same todo as in https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/public/js/views/services/plugins/ez-contenttreeplugin.js#L62 for instance so that when it's supported we can easily find relevant places where this has to be put in place.